src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+
+  sourceString
+    .split(';')
+    .filter((item) => item.includes(':'))
+    .forEach((item) => {
+      const [property, ...valueParts] = item.split(':');
+      const value = valueParts.join(':').trim();
```

Comment:
The current implementation splits the string by the first colon, which can cause issues if the value contains additional colons, such as in URLs. Consider using a regular expression or a different approach to correctly split the property and value.